### DISCUSSION TITLE
Disable focus on Error

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [react-hook-form/react-hook-form#6113] [Q&A] [answered]
When i submitting form with error it's focusing input field..... how can i disable focus ?
without focus i want to show error message

**Comment by bluebill1049** [comment, marked as answer]
read the doc: https://react-hook-form.com/api/useform `shouldFocusError`

**Reply by MuttakinHasib** [reply]
how can I block form submission when I setError manually ?
